style.css

```diff
@@ -1,18 +1,752 @@
-/*@import "fonts.css";*/
-
-/*body {
-	font-family: "Montserrat-SemiBold";
-}*/
-
 .header, .start, .specifications, .salon, .description__title, .parameters__item, .safety__title, .stars__subtitle, .fivestars__textstars{
-	font-family: "Montserrat-SemiBold";
+font-family: "Montserrat-SemiBold";
 }
 .specifications__item, .description__link, .stars__item, .fivestars__text, .fivestars__link{
-	font-family: "Montserrat-Bold";
+font-family: "Montserrat-Bold";
 }
 .description__text, .safety__text{
-	font-family: "Montserrat-Medium";
+font-family: "Montserrat-Medium";
 }
 .parameters__small{
-	font-family: "Montserrat-Regular";
+font-family: "Montserrat-Regular";
+}
+
+
+body{
+max-width: 1440px;
+margin: 0 auto;
+display: flex;
+flex-direction: column;
+}
+
+.header{
+display: flex;
+max-width: 1440px;
+height: 51px;
+padding-top: 17px;
+justify-content: center;
+align-content: center;
+align-items: center;
+justify-content: space-between;
+}
+
+.menu{
+font-size: 1vw;
+display: flex;
+justify-content: center;
+align-content: center;
+}
+
+.menu__store{
+display: flex;
+justify-content: center;
+align-content: center;
+bottom: 20px;
+}
+
+.menu__link{
+min-width: 65px;
+display: flex;
+align-items: center;
+align-content: center;
+font-weight: 600;
+font-size: 15px;
+line-height: 18px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html_rus3/pullRequest?prid=6047524)